source/FreeRTOS_IP_Utils.c

```diff
@@ -1113,6 +1113,8 @@ const char * FreeRTOS_strerror_r( BaseType_t xErrnum,
 
         default:
             /* Using function "snprintf". */
+            /* exception used for logging purposes only */
+            /* coverity[misra_c_2012_rule_21_6_violation] */
```

Comment:
Rule 21.6 is ridiculous. We should just use a global deviation for it and ensure that Rules 22.X are enabled. 

I think a good justification regarding printf usage is that "-Wformat -Werror=format-security" is used when compiling, which should prevent format string errors.